src/txn/txn_rollback_to_stable.c

```diff
@@ -91,7 +93,23 @@ __rollback_abort_update(WT_SESSION_IMPL *session, WT_ITEM *key, WT_UPDATE *first
         if (upd->txnid == WT_TXN_ABORTED)
             continue;
 
-        if (rollback_timestamp < upd->durable_ts || upd->prepare_state == WT_PREPARE_INPROGRESS) {
+        /*
+         * The unstable update according to the following needs to be aborted.
+         * 1. An invisible update according to the checkpoint snapshot during recovery.
+         * 2. The update durable timestamp is greater than the stable timestamp.
+         * 3. A prepared update.
```

Comment:
I suggest: 3. The update is a prepared update.

Reply:
Updated as per the suggestion.